substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -30,46 +30,49 @@
 //! ## Typical use-cases.
 //! ##### Get default config from runtime.
 //!
-//!	Query the default genesis config from the provided `runtime.wasm` and use it in the chain
-//! spec. Tool can also store runtime's default genesis config in given file:
-//!	```text
-//! chain-spec-builder create -r runtime.wasm default /dev/stdout
+//! Query the default genesis config from the provided `runtime.wasm` and use it in the chain
+//! spec. The tool allows specifying where to write the chain spec, and optionally also where the
+//! write the default genesis state config (which is `/dev/stdout` in the following example):
+//! ```text
+//! chain-spec-builder --chain_spec_path ./my_chain_spec.json create -r runtime.wasm default /dev/stdout
 //! ```
-//! 
-//! _Note:_ [`GenesisBuilder::create_default_config`][sp-genesis-builder-create] runtime function is called.
+//!
+//! _Note:_ [`GenesisBuilder::create_default_config`][sp-genesis-builder-create] runtime function is
+//! called.
 //!
 //!
 //! ##### Generate raw storage chain spec using genesis config patch.
 //!
 //! Patch the runtime's default genesis config with provided `patch.json` and generate raw
 //! storage (`-s`) version of chain spec:
-//! ```text
-//! chain-spec-builder create -s -r runtime.wasm patch patch.json
+//!/
+/! chain-spec-builder create -s -r runtime.wasm patch patch.json
```

Comment:
Is it intentional? Seems like sth is broken here.

Reply:
Oops, yep fixed.